cfme/services/catalogs/cloud_catalog_item.py

```diff
@@ -157,3 +148,59 @@ def create(self):
         })
         sel.click(template_select_form.add_button)
         flash.assert_success_message('Service Catalog Item "{}" was added'.format(self.name))
+
+
+@navigator.register(CloudCatalogItem, 'All')
+class ItemAll(CFMENavigateStep):
```

Comment:
This would introduce a second. @sshveta and I discussed this offline, and determined that the existing navigation classes in catalog_item.py are identical, and its only the cloud-type forms that are different.  The CRUD functions need to be unique, or contain the logic to handle the different catalog item types.